### ISSUE TITLE
Inserts closing brace on wrong line

### ISSUE THREAD

**Issue by cipri-tom** [randy3k/LaTeXYZ#12]
Hi,

Thank you for the package ! I can't believe I've only discovered it now! All features are amazing and it fixes my biggest annoyance, namely it completes `\(` with `\)` by default ❤️ . 

I think I have discovered a bug. I am using it with LaTeXTools, standard install, no custom config for any.

**Example** (`|` shows cursor position)
```latex
\begin{minipage}|
	
\end{minipage}
```
Press `{`

**Expected behaviour:** The matching `}` is inserted on the same line, resulting in `\begin{minipage}{|}`

**Actual behaviour**: The matching `}` is inserted on the next line, at best:
```latex
\begin{minipage}{|
}

\end{minipage}
```

 or on some further line with more complex examples:
```latex
\begin{figure}
	\begin{minipage}{|
		
	\end{minipage}
	\includegraphics[width=.5\linewidth]{1}
	\includegraphics[width=.5\linewidth]{2}
        } % TA-DAA the matching brace for minipage 2nd arg was inserted here
\end{figure}
```

If I disable `LateXYZ`, this behaviour no longer appears.

**Comment by cipri-tom** [comment]
Thank you for the quick reply !! 

So, in the **Example** above (just a minipage, cursor at end), the command is:
`wrap_block {"begin": "{", "end": "}"}`. That's the only command. And the `}` ends up after `\end{minipage}`.

Wait, I'm just realising this is different from what I wrote. Still wrong, but different.

It turns out I was indented twice (2 tabs) for the above. I decreased the indent to 1 tab, for the whole block (the pair `begin, end`), and it works correctly, with the command being: `latexyz_insert_snippet {"contents": "{$0}", "create_fields": true}`

I further decrease the indent, so the `\begin{minipage}` is now at the first column, and I experience the first behaviour: `}` is on the line after `\begin{minipage}{`. The command is `wrap_block {"begin": "{", "end": "}"}`.

I believe a default latex snipped takes precedence over the one from `LateXYZ`.
